### ISSUE TITLE
Getting Started with CNOE (quickstart)

### ISSUE THREAD

**Issue by csantanapr** [cnoe-io/website#38]
Got some feedback from new users that they don't know how to get started quickly to get something up and running.

Now that we have idpbuilder we should have it easier to find, install, and get up and running from the front page

**Comment by csantanapr** [comment]
It is difficult to find how to install idpbuilder, user was not able to find the idpbuilder page and from there it was not obvious how to install idpbuilder it was a grey out note on the page 
![image](https://github.com/cnoe-io/website/assets/1094878/38b9fdc6-9186-4c4c-9944-ab4d919ba759)

**Comment by nimakaviani** [comment]
agreed. just had a discussion about this with @cbleach ... will bring on a designer to help with the website soon.
